app/src/main/java/com/concordium/wallet/ui/cis2/SendTokenActivity.kt

```diff
@@ -44,6 +44,7 @@ class SendTokenActivity : BaseActivity() {
         const val ACCOUNT = "ACCOUNT"
         const val TOKEN = "TOKEN"
         const val SEND_ONLY_SELECTED_TOKEN = "SEND_ONLY_SELECTED_TOKEN"
+        const val FLOW_STATE = "FlowFinished"
```

Comment:
The name of this can I be more descriptive? Like `TOKEN_TRANSFER_FLOW_FINISHED`

Reply:
sure.